### DISCUSSION TITLE
Heads Up Breaking changes in .NET 8

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#3507] [General]
Title: Exploring .NET Core 8.0 Compatibility - A Quick Overview

Hey fellow developers!

I recently stumbled upon the compatibility guide for .NET Core 8.0, and I thought I'd share my findings with you all. The guide, available at [Microsoft's .NET Core Compatibility page](https://learn.microsoft.com/en-us/dotnet/core/compatibility/8.0), provides a comprehensive look at what's compatible and what's not in the latest version.

**Comment by sbwalker** [comment]
Just to be clear so that developers are not overly alarmed by this title of this discussion post.... 

The Oqtane Framework itself did NOT experience any compatibility issues or breaking changes when upgrading to .NET 8. In addition, there were NO breaking changes in the Oqtane Framework which will affect modules or themes.

However it is definitely possible that you may experience compatibility issues with other .NET applications when you attempt to upgrade them. Thank you for sharing @leigh-pointer
